_dashboards/index.md

```diff
@@ -11,15 +11,56 @@ redirect_from:
 
 {%- comment -%}The `/docs/opensearch-dashboards/` redirect is specifically to support the UI links in OpenSearch Dashboards 1.0.0.{%- endcomment -%}
 
-# OpenSearch Dashboards
+# Getting started with OpenSearch Dashboards
 
-OpenSearch Dashboards is the default visualization tool for data in OpenSearch. It also serves as a user interface for many of the OpenSearch plugins, including security, alerting, Index State Management, SQL, and more.
+OpenSearch Dashboards is the preferred visualization platform for analyzing data in OpenSearch. It serves as the user interface for many openSearch features, including security, alerting, index state management, SQL, and more.
 
+Out-of-the-box OpenSearch Dashboards includes several tools for discovering and visualizing data, and OpenSearch provides a full suite of plugins that you can use to monitor events, identify trends, and automate recurring activities.
 
-## Get started with OpenSearch Dashboards
+![Sample dashboards](../images/dashboards-example.png)
 
-1. After starting OpenSearch Dashboards, you can access it at port 5601. For example, http://localhost:5601.
-1. Log in with the default username `admin` and password `admin`.
-1. Choose **Try our sample data** and add the sample flight data.
-1. Choose **Discover** and search for a few flights.
-1. Choose **Dashboard**, **[Flights] Global Flight Dashboard**, and wait for the dashboard to load.
+# Try OpenSearch Dashboards
+You can try OpenSearch Dashboards' features in the [OpenSearch Dashboards Playground](https://playground.opensearch.org/app/home).
+
+Choose **OpenSearch Dashboards > Dashboards > Create dashboard** and explore the sample Global Flight, Web Traffic, and Revenue dashboards.
+
+![Dashboards playground screenshot](../images/dashboards-playground.png)
```

Comment:
You cannot use relative paths in links to our site. Use full path with a dynamic site url, as Jeff suggested.

Reply:
I'm questioning if this image adds any information. On the one hand, it's a visual, so it can draw attention. On the other hand, this is exactly what they will see if they go to the playground link... @hdhalter - could you weigh in?